### ISSUE TITLE
Ideas for cell types

### ISSUE THREAD

**Issue by paulelms** [HappenApps/Quiver#363]
It may be possible to create pluggable cells. I wish cells with taskpaper format (plain text todo list). Will be cool to implement script that will print all `@today` tasks from all taskpaper cells, etc.
Also be cool to have sketches (like in apple notes). And attached/linked mindmaps from iThoughts/MindNode/etc. displayed in preview mode.

**Comment by hlissner** [comment]
I'd like to build on this suggestion:

Because this is a note-taking app for programmers, why not have a "script" cell-type? You'd specify a path to a script and its output will fill the cell (whenever the path changes, an 'update' button is pressed, or at whatever interval seems reasonable to you).

For instance, 

```js
{
  "title": "Taskpaper todo list",
  "cells": [
    {
      "type": "script",
      "path": "~/bin/get_todos.sh",
      "output": "markdown", // how the output is interpreted; accepts any cell type
      "data": "..."
    }
  ]
}
```

`path` could point to any script with a valid shebang line, so that scripts may be written in any language.

**Comment by eddyg** [comment]
+1

I'd also like to see a TaskPaper cell type.

**Comment by cheslip** [comment]
+1

TaskPaper would be perfect for Quiver to keep notes and todos all in one place.

**Comment by weezard** [comment]
+1 this will make Quiver infinitely usable and extensible. I love the mindmap idea

**Comment by sterlingdax** [comment]
- Yes, I too would like Taskpaper support @celltypes

**Comment by helmut72** [comment]
Yes, at least Taskpaper, but sketches like in Apple Notes (and Pencil support in iOS) would be also cool.

**Comment by keeprock** [comment]
+1 for MindNode

**Comment by keroro520** [comment]
+1 for MindNode

Mind-map is also a important type of content.

**Comment by n0cturnuz** [comment]
+1
